### PR TITLE
feat: add platform switch to library

### LINKED ISSUE #222: Add platform switch
Integrate https://github.com/graasp/graasp-ui/issues/215 into Library

### PR DESCRIPTION
**Pull request by dialexo**
Closes #222 

Q: should we harmonize the navigation between the platforms at some point? Currently in Library, the spacing on the left of the logo is not the same as the other platforms (since the sidebar controls are absent), and adding artificial margins would look weird

[Kapture 2023-02-13 at 12.09.27.webm](https://user-images.githubusercontent.com/14943421/218443335-0f6aa026-6d28-43fb-9ed1-30f4736af80b.webm)

### CODE REVIEW

File: src/components/common/HeaderNavigation.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+  color: 'inherit',
+  display: 'flex',
+  alignItems: 'center',
+};
+
+const StyledLink = styled(Link)(() => linkStyles);
+
+interface HeaderNavigationProps {
+  rootId?: string;
+  topItemName?: string;
```
Comment (spaenleh) [inline src/components/common/HeaderNavigation.tsx:116]: What do you want to use this prop for ? It does not seem to get a value where this component is called.

**Review by spaenleh — APPROVED** [review]
🚢 LGTM ! Thank you for this, it looks super cool !

File: src/components/common/HeaderNavigation.tsx (hunk 1 above)
Comment (dialexo) [reply src/components/common/HeaderNavigation.tsx]: Good catch! Oops it was a copy paste from the component in Builder, but the prop is not used in LIbrary  👀

File: src/components/collection/Collection.js [hunk 2]
```diff
@@ -115,26 +118,32 @@ const Collection = ({ id }) => {
...
-          </Alert>
-        )}
+        {
+          // show alert only if 1. user is logged in, 2. it has at least read access and 3. item is not published
+          currentMember?.id && canRead && !isPublished && (
+            <Alert severity="warning">
+              You are viewing this item in Library preview mode. It cannot be
+              viewed publicly.
+              {
+                // if the user is the admin of the item, also suggest publishing from Buider
```
Comment (spaenleh) [inline src/components/collection/Collection.js:128]: Nitpick:
```suggestion
                // if the user is the admin of the item, also suggest publishing from Builder
```

**Review by spaenleh — APPROVED** [review]
🦅 LGTM! Thank you for the update !

**Pull request merged**